VERSIONING.md

```diff
@@ -1,10 +1,60 @@
 Versioning and Branching in Konveyor Projects
 =============================================
 
-This applies to the following Konveyor Projects:
+This document outlines how we handle versioning of our projects in the Konveyor
+organization (Konveyor projects).
```

Comment:
Wonder if worth considering a word change from 'projects' to 'components' as Konveyor has moved away from being a grouping of independent projects and is now aligned to the [Unified Vision ](https://github.com/konveyor/enhancements/tree/master/enhancements/unified_experience) as being a single project.

I thought continued use of referring to Konveyor as separate projects may be confusing, yet I'm not positive.